### PR TITLE
Fix-76315 add *.podspec to file associations 

### LINKED ISSUE #76315: add *.podspec to file associations
*.podspec files are just ruby. I can add them in manually ofcourse. It would be nice if vscode had an entry by default for it so everyone gets syntax highlighting for it out of the box.

Thank you.

### PR DESCRIPTION
**Pull request by skprabhanjan**
@alexr00 , I added .podspec also as an extension so that those files are automatically detected and fixes #76315 .
Please review this.

This is how sample .podspec files looks now.
FYI : @graph.

image

### CODE REVIEW

File: extensions/ruby/package.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
 		"languages": [{
 			"id": "ruby",
-			"extensions": [ ".rb", ".rbx", ".rjs", ".gemspec", ".rake", ".ru", ".erb" ],
+			"extensions": [ ".rb", ".rbx", ".rjs", ".gemspec", ".rake", ".ru", ".erb",".podspec" ],
 			"filenames": [ "rakefile", "gemfile", "guardfile", "podfile", "capfile" ],
 			"aliases": [ "Ruby", "rb" ],
 			"firstLine": "^#!\\s*/.*\\bruby\\b",
 			"configuration": "./language-configuration.json"
-		}],
+    }],
```
Comment (alexr00) [inline extensions/ruby/package.json]: nit: Can you fix the spacing on this line?

**Review by alexr00 — CHANGES_REQUESTED** [review]

File: extensions/ruby/package.json (hunk 1 above)
Comment (skprabhanjan) [reply extensions/ruby/package.json]: Sorry my bad , Done now :)

**Comment by skprabhanjan** [comment]
Changes done :)

**Review by alexr00 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**